Packs/Dropbox/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Dropbox",
     "description": "Use the Dropbox integration to fetch events",
     "support": "xsoar",
-    "currentVersion": "1.1.6",
+    "currentVersion": "1.1.7",
     "author": "Cortex XSOAR",
     "url": "https://www.paloaltonetworks.com/cortex",
     "email": "",
```

Comment:
need to add tags, keywords and useCases as needed.
https://xsoar.pan.dev/docs/documentation/pack-docs#pack-keywords-tags-use-cases--categories